Sources/_StringProcessing/Engine/MEBuiltins.swift

```diff
@@ -1,13 +1,221 @@
-
+@_implementationOnly import _RegexParser // For AssertionKind
 
 extension Processor {
+  mutating func matchBuiltin(
+    _ cc: _CharacterClassModel.Representation,
+    _ isInverted: Bool,
+    _ isStrictAscii: Bool
+  ) -> Bool {
+    guard let c = load() else {
+      signalFailure()
+      return false
+    }
+
+    var matched: Bool
+    var next = input.index(after: currentPosition)
```

Comment:
It seems like we can reduce the complexity a little bit by handling `.anyScalar` (and possibly others) early, and that would have fewer live variable across the switch.